Dockerfile

```diff
@@ -29,6 +29,7 @@ RUN apt-get update && apt-get -qy install --no-install-recommends \
  libmysqlclient-dev \
  libssl-dev \
  python3-dev \
+ build-essential \
```

Comment:
this apt package includes the `make` command, which is useful when you want to `make requirements` inside the `app-shell` (inside the docker container).